frontend/src/pages/RegistrationPage.jsx

```diff
@@ -1,30 +1,45 @@
-import { useContext, useState } from "react";
+import { useState } from "react";
 import { useNavigate } from "react-router-dom";
-import { AuthContext } from "../contexts/AuthContext";
-import Cookies from "js-cookie";
 import { register } from "../api/auth";
+import css from "./LoginAndRegisterPage.module.css";
 
 export default function RegistrationPage() {
   const navigate = useNavigate();
   const [username, setUsername] = useState("");
   const [password, setPassword] = useState("");
-  const { checkAuth } = useContext(AuthContext);
+  const [matchPassword, setMatchPassword] = useState("");
   const handleSubmit = async e => {
     e.preventDefault();
-    await register({ username, password });
-    const isAuth = checkAuth(Cookies.get("user"));
-    if (isAuth) {
+    if (password !== matchPassword) {
```

Comment:
Please remove the console log, and add some indication to the user that the passwords don't match. Consult Roma if you're not sure how to do it better design-wise.